packages/evolution-frontend/src/components/pageParts/Survey.tsx

```
@@ -195,7 +197,7 @@
                         submitted={submitted}
                     />
                 </form>
-                <div dangerouslySetInnerHTML={{ __html: t(['survey:footer', 'main:footer']) }} />
+                <div dangerouslySetInnerHTML={{ __html: t(['survey:footer', 'main:footer'], { context: appContext }) }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)